android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -90,6 +94,21 @@
         void onLabelPick(LabelPickResult labelPickResult, float positionX, float positionY);
     }
 
+    /**
+     * Interface for a callback to receive the picked {@link Marker}
+     * Triggered after a call of {@link #pickMarker(float, float)}
+     * Listener should be set with {@link #setMarkerPickListener(MarkerPickListener)}
+     */
+    public interface MarkerPickListener {
+        /**
+         * Receive the picked {@link Marker}
+         * @param marker The {@link Marker} the marker that has been selected
+         * @param positionX The horizontal screen coordinate of the tapped location
```

Comment:
Should we update the description "The horizontal screen coordinate of the tapped location" to "The horizontal screen coordinate of the picked location" since `pickMarker(...)` can be invoked at any time even on non-tap events?